utils/exp_manager.py

```diff
@@ -607,13 +609,24 @@ def objective(self, trial: optuna.Trial) -> float:
         # Account for parallel envs
         optuna_eval_freq = max(optuna_eval_freq // model.get_env().num_envs, 1)
         # Use non-deterministic eval for Atari
-        eval_callback = TrialEvalCallback(
-            eval_env,
-            trial,
-            n_eval_episodes=self.n_eval_episodes,
-            eval_freq=optuna_eval_freq,
-            deterministic=self.deterministic_eval,
-        )
+        if self.logging != "":
```

Comment:
please use the if/else only for defining the variables that change (clearer to see the difference) so you have only one call to the constructor.